gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -793,6 +910,73 @@ public static LicenseOption fields(LicenseField... fields) {
   }
 
   /**
+   * Class for specifying operation get options.
+   */
+  class OperationOption extends Option {
```

Comment:
Just curious: this naming is consistent within Compute and also with BigQuery, but in storage we include "get" (eg BlobGetOption).  Is the rationale for this difference that we want shorter class names (so remove "get" where possible), but because there are many types of Blob Options (BlobTargetOption, BlobWriteOption, etc), it's clearer to leave "get" in the class name for storage?